pkg/server/middleend.go

```diff
@@ -121,7 +136,7 @@ func (s *Server) UpdateEncryptedVolume(ctx context.Context, in *pb.UpdateEncrypt
 	}
 	params2 := AccelCryptoKeyCreateParams{
 		Cipher: r.FindStringSubmatch(in.EncryptedVolume.Cipher.String())[1],
-		Name:   "super_key2",
+		Name:   "super_key",
 		Key:    string(in.EncryptedVolume.Key),
 		Key2:   strings.Repeat("b", len(in.EncryptedVolume.Key)),
```

Comment:
Should this be "a"'s repeated?

Reply:
wanted to show a change from `a` to `b` since this is `update` command